ci-operator/step-registry/servicemesh/istio/e2e/arm/servicemesh-istio-e2e-arm-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: servicemesh-istio-e2e-arm
```

Comment:
Do we need this extra workflow? I'm not seeing where it's different from the existing one, I mean, where do we specify we are creating ARM clusters in this workflow?

Reply:
Deleted the arm profile that where there because I initially did a test with hypershift. Now the variables will be overridden from the job definition and the same profile will be used